packit/api.py

```diff
@@ -2343,11 +2347,27 @@ def run_osh_build(
         if csmock_args:
             cmd.append("--csmock-args=" + shlex.quote(csmock_args))
 
-        cmd.append("--config=" + str(chroot))
+        osh_options = self.package_config.osh_options
+
+        if osh_options is not None:
+            analyzer = osh_options.get("analyzer")
+            config = osh_options.get("config")
+            profile = osh_options.get("profile")
+
+        if analyzer:
+            cmd.append("--analyzer=" + shlex.quote(analyzer))
+        if profile:
+            cmd.append("--profile=" + shlex.quote(profile))
```

Comment:
```suggestion
        if analyzer := osh_options.get("analyzer"):
            cmd.append("--analyzer=" + shlex.quote(analyzer))
        if profile := osh_options.get("profile"):
            cmd.append("--profile=" + shlex.quote(profile))
```

Afterwards you can remove the declarations all the way at the top, they're no longer needed